Libraries/Components/Button.windows.js

```diff
@@ -116,6 +143,32 @@ class Button extends React.Component {
     );
     const formattedTitle = Platform.OS === 'android' ? title.toUpperCase() : title;
     const Touchable = Platform.OS === 'android' ? TouchableNativeFeedback : TouchableOpacity;
+    var content =
```

Comment:
>var [](start = 4, length = 3)

Can you update all usages of `var` to `let` or `const`. `let` and `cost` are preferred over `var` because they are block-scoped rather than function-scoped

Reply:
fixed

---
In reply to: [156556993](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r156556993) [](ancestors = 156556993)